dts/bindings/clock/nordic,nrf53-hfxo.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nordic nRF high-frequency crystal oscillator (nRF53 series)
+
+compatible: "nordic,nrf53-hfxo"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
```

Comment:
I think it would be beneficial to introduce common compatible like `nordic,nrf53-clock.yaml`, that would contain common `clock-frequency` and `load-capacitors` properties, to be utilized by both `nrf53-hfxo` & `nrf53-lfxo`.
`clock-frequency` could be extracted to even more generic `nordic-nrf5x-clock.yaml` . This way we would increase number of bindings, but reduce duplication across bindings

Reply:
`clock-frequency` is already common (comes from `fixed-clock`), so the only one that can be made common is `load-capacitors`.